src/elements/content-sidebar/hooks/useSidebarMetadataFetcher.ts

```diff
@@ -174,6 +205,37 @@ function useSidebarMetadataFetcher(
         [api, file, onApiError],
     );
 
+    const extractSuggestions = React.useCallback(
+        async (templateKey: string, fields: MetadataTemplateField[]) => {
+            const aiAPI = api.getIntelligenceAPI();
+            let answer = {};
+            try {
+                answer = await aiAPI.extractStructured({
+                    items: [file],
+                    fields,
+                });
+            } catch (error) {
+                throw new ElementsError(error.message, ERROR_CODE_FETCH_METADATA_SUGGESTIONS);
+            }
+
+            if (isEmpty(answer)) {
+                throw new ElementsError('No suggestions found.', ERROR_CODE_EMPTY_METADATA_SUGGESTIONS);
```

Comment:
Might be out of scope for this PR but realized we have a some repeat Error messages across the Metadata files. We should consolidate all the error messages into a constants file so we don't have typos or inconsistencies.

Reply:
The only other occurrence of `No suggestions found.` is in [`getMetadataSuggestions`](https://github.com/box/box-ui-elements/blob/fa89f876d54680b95bdc2fd3eb0d07a3c729f03e/src/api/Metadata.js#L1053) that is as far as I know not used.
I will confirm with the team if this function should be deleted in upcoming PRs.